exec/exec_test.go

```diff
@@ -408,3 +409,61 @@ func TestNonSpec(t *testing.T) {
 func TestSpec(t *testing.T) {
 	testModules(t, specTestsDir)
 }
+
+func loadModuleFindFunc(t *testing.B, fileName, funcName string, nativeBackend bool) (*exec.VM, uint32) {
+	file, err := os.Open(fileName)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer file.Close()
+
+	module, err := wasm.ReadModule(file, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err = validate.VerifyModule(module); err != nil {
+		t.Fatalf("%s: %v", fileName, err)
+	}
+
+	vm, err := exec.NewVM(module, exec.EnableAOT(nativeBackend))
+	if err != nil {
+		t.Fatalf("%s: %v", fileName, err)
+	}
+	return vm, module.Export.Entries[funcName].Index
+}
+
+func BenchmarkU64Arithmetic10Interpreted(b *testing.B) {
+	vm, funcIndex := loadModuleFindFunc(b, "testdata/rust-basic.wasm", "loopedArithmeticI64Benchmark", false)
+	for n := 0; n < b.N; n++ {
+		vm.ExecCode(int64(funcIndex), 10, 10)
```

Comment:
one should probably store one of the 2 returned values into a global variable to prevent unwanted optimization from the compiler.

Reply:
Ack, will do in morning :)